modules/plugin/svg/src/main/java/org/geotools/renderer/style/SVGGraphicFactory.java

```diff
@@ -117,31 +117,9 @@ public SVGGraphicFactory(Map<Key, Object> hints){
             }
         }
         else {
-            this.resolver = defaultResolver();
```

Comment:
Is the above code necessary - XMLHandlerHints#toEntityResolver seems to be doing the same thing.
(note - this.implementationHints.put( Hints.ENTITY_RESOLVER, this.resolver ); is different)